x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -0,0 +1,126 @@
+[role="xpack"]
+[[fips-140-compliance]]
+== FIPS 140-2
+
+The Federal Information Processing Standard (FIPS) Publication 140-2, (FIPS PUB
+140-2), titled "Security Requirements for Cryptographic Modules" is a U.S.
+government computer  security standard used to approve cryptographic modules.
+{es} is FIPS-140 compliant and as such can run in a FIPS-140 enabled JVM. In
```

Comment:
maybe instead of `is FIPS-140 compliant` we can say `offers a a FIPS 140-2 compliant mode`